module-system/module-implementations/sov-blob-storage/src/capabilities.rs

```diff
@@ -107,22 +146,7 @@ impl<C: Context, Da: DaSpec> BlobSelector<Da> for BlobStorage<C, Da> {
             // Gas metering suppose to prevent saving blobs from not allowed senders if they exit mid-slot
             let to_defer: Vec<&Da::BlobTransaction> = to_defer
                 .iter()
-                .filter(|b| {
-                    let is_allowed = self
-                        .sequencer_registry
-                        .is_sender_allowed(&b.sender(), working_set);
-                    // This is the best effort approach for making sure,
-                    // that blobs do not disappear silently
-                    // TODO: Add issue for that
-                    if !is_allowed {
-                        info!(
-                            "Blob hash=0x{} from sender {} is going to be discarded",
-                            hex::encode(b.hash()),
-                            b.sender()
-                        );
-                    }
-                    is_allowed
-                })
+                .filter(|b| self.filter_by_sender(b, working_set))
```

Comment:
Nit: Maybe call it `filter_by_allowed_sender`

Reply:
Done.